troubleshooting/nvidia-troubleshooting.md

```diff
@@ -71,3 +78,70 @@ If you see a terminal window in the top left corner, it means you most likely su
 
 1.  Reboot and let the system boot from the normal boot configuration. You should be able to use X under Xen now.
 
+
+Disabling Nouveau
+---------------------
```

Comment:
This is a bit of a paradoxical situation I believe as they wouldn't see an error without a built-in graphics card when nouveau decides to crash.

I'll add it if you believe I should.